libminifi/src/core/ProcessGroup.cpp

```diff
@@ -153,8 +153,10 @@ void ProcessGroup::startProcessingProcessors(const std::shared_ptr<TimerDrivenSc
   for (const auto processor : failed_processors_) {
     try {
       processor->onUnSchedule();
+    } catch (const std::exception& ex) {
+      logger_->log_error("Exception occured during unscheduling processor: %s (%s), type: %s, what: %s", processor->getUUIDStr(), processor->getName(), typeid(ex).name(), ex.what());
```

Comment:
This log line is useful, but I wonder if it can occur... Throw in unschedule definitely sounds like something to avoid.

Reply:
Probably shouldn't occur, but I'd rather leave it in, to be safe. Who knows what can happen in processors, especially third-party ones.